isilon_create_users.sh

```diff
@@ -260,7 +311,7 @@ for group in $SUPER_GROUPS; do
         user="$user$CLUSTER_NAME"
         isi auth groups modify $group --add-user $user --zone $ZONE
         [ $? -ne 0 ] && addError "Could not add user $user to $group group in zone $ZONE"
-        sprgrp="$sprgrp,$user"
+        sprgrp="$sprgrp$user,"
```

Comment:
I'm not convinced this is the long-term solution, but it's no worse than what we currently do. Just a note...